.github/workflows/main.yml

```diff
@@ -11,6 +11,19 @@ on:
       - '**'
 
 jobs:
+  macos:
+    runs-on: macOS-latest
```

Comment:
I suggest that `macOS-10.14` would be better because it ensures that Github Action will run on Xcode 10.3 in macOS 10.14. On the other hand, the latest means 10.15 and Xcode 10.11

Ref: https://help.github.com/en/github/automating-your-workflow-with-github-actions/software-in-virtual-environments-for-github-actions#installed-sdks